documentation/reference/document-select-language.html

```diff
@@ -178,7 +186,45 @@ <h3 id="comparison">Comparison</h3>
 </table>
 </p>
 
-<h4>Locale / Character sets</h4>
+<h4 id="null-value-comparisons">Comparisons with missing fields (null values)</h4>
+<p>
+The only comparison operators that are well-defined when one or both operands may be <code>null</code>
+(i.e. field is not present) are <code>==</code> and <code>!=</code>. Using any other comparison operators
+on a <code>null</code> value will yield a special invalid value.
+</p>
+<p>
+Invalid values may "poison" any logical expression they are part of:
+</p>
+<ul>
+  <li><code>AND</code> returns invalid iff none of its operands are false and at least one is invalid</li>
```

Comment:
iff ?

Reply:
"if and only if"

Reply:
oh ...

Reply:
ohh that is

Reply:
I see we don't use "iff" anywhere else in our documentation so I'll just use "if" instead to avoid any confusion. This isn't a formal behavior spec, so doesn't need that kind of level of detail.